mesonbuild/wrap/wrap.py

```diff
@@ -261,6 +265,31 @@ def get_svn(self, p):
             subprocess.check_call(['svn', 'checkout', '-r', revno, p.get('url'),
                                    p.get('directory')], cwd=self.subdir_root)
 
+    def get_maven(self, p):
+        if not os.path.isdir(self.cachedir):
+            os.mkdir(self.cachedir)
+
+        ofname = os.path.join(self.cachedir, p.get('artifact'))
+        if os.path.exists(ofname):
+            mlog.log('Using', mlog.bold(p.get('artifact')), 'from cache.')
+        else:
+            group, artifact, version = p.get('artifact').split(':')
+            baseurl = '/'.join([p.get('repository')] + group.split('.') + [artifact] + [version])
+            jarname = artifact + '-' + version + '.jar'
+            srcurl = baseurl + '/' + jarname
+            mlog.log('Downloading', mlog.bold(p.get('artifact')), 'from', mlog.bold(srcurl))
+            dhash, jarfile = self.get_data(srcurl)
+
+        target_dir = os.path.join(self.subdir_root, p.get('directory'))
+        if not os.path.exists(target_dir):
+            os.mkdir(target_dir)
+        shutil.copyfile(jarfile, os.path.join(target_dir, jarname))
+        with open(os.path.join(target_dir, 'meson.build'), 'w') as file:
+            file.write('project(\'' + artifact + '\', \'java\')\n')
+            file.write('javac = meson.get_compiler(\'java\')\n')
+            #file.write(artifact + '_dep = javac.find_library(\'' + jarname + '\')')
```

Comment:
__[Flake8]__

[E265] block comment should start with '# '


<!-- Comment by [SideCI](https://sideci.com) -->